EasyPost/Models/API/Form.cs

```diff
@@ -1,26 +1,20 @@
-﻿using System;
+﻿using EasyPost._base;
 using Newtonsoft.Json;
 
-namespace EasyPost
+namespace EasyPost.Models.API
 {
-    public class Form : Resource
+    public class Form : EasyPostObject
     {
         #region JSON Properties
 
-        [JsonProperty("created_at")]
-        public DateTime? created_at { get; set; }
         [JsonProperty("form_type")]
         public string form_type { get; set; }
         [JsonProperty("form_url")]
         public string form_url { get; set; }
-        [JsonProperty("id")]
-        public string id { get; set; }
         [JsonProperty("mode")]
-        public string mode { get; set; }
+        public new string mode { get; set; }
```

Comment:
Issue: I believe this is a typo? We don't want to be making a new mode here? Also, isn't mode inherited?